### ISSUE TITLE
Github link and weird personal text in web page (Usage of Hydejack as remote_theme)

### ISSUE THREAD

**Issue by ChristopherA** [qwtel/hydejack#117]
I'd prefer not to fork Hydejack and instead use it natively with Github's remote_theme capability. I don't want to use Gem or Ruby on my local machine — just truly take advantage of what remote_theme's can offer. Thus I'd like to minimize the files in my repo to just the essentials to support a small amount of mostly non-blog content.

I have a very simple repo up at https://github.com/BlockchainCommons/SmartCustody which serves https://www.smartcustody.com/ but I'm having some odd problems. I've been able to work around most of them (for instance, putting img assets in root rather then in the /assets folder), but this one is more serious and I have no idea what is going on.

Basically, you will see a github icon showing up in the sidebar, with a link to `https://github.com/%7B`. I have social turned off (turning it on makes no different). If you inspect the source of my page, you'll see right after that link is several pages of my personal github info (my repos, etc.). I'm guessing something when I use remote_theme is stuffing that info into the web page.

Any ideas?
